pyblish_lite/window.py

```diff
@@ -1071,7 +1071,7 @@ def heads_up(self, title, message, command=None):
     def paintEvent(self, event):
         if Qt.__qt_version__.startswith("5") and os.name == "nt":
             # Only tested with Windows DPI scaling
-            scale = self.windowHandle().screen().logicalDotsPerInch() / 96.0
+            scale = self.window().windowHandle().screen().logicalDotsPerInch() / 96.0
```

Comment:
Eeeek! There were already too many things that could go wrong with this line, could we break this up and handle each one separately? Especially `if window is None:` and `if window_handle is None` and `if screen is None`.

Reply:
Updated the PR, as in the issue... there also would be `widget.nativeParentWidget()` but the method potentially returns 0...
I asked in the PySide gitter channel but there was not much response. I am using widget.window() in many places and at least in my constructs it always gets the correct topLevelWidget I want. 
As I said I dont know in which scenario the window method would not return a native widget...